wiki/usecases/crudPermissions.md

```diff
@@ -43,4 +44,7 @@ tbd
 tbd
 
 # 6. Function Points
-tbd
\ No newline at end of file
+tbd
+
+
+For feedback about this page go to this [issue](https://github.com/FileFighter/filefighter.github.io/issues/17){:target="_blank"}.
```

Comment:
You know you can add `issuelink: https://github.com/FileFighter/filefighter.github.io/issues/17` to the header of the file to automaticly generate the this sentence. i mean except you changed that. :D